packages/cyberstorm/src/components/AntiResult/index.ts

```diff
@@ -0,0 +1,4 @@
+export { Root } from "./AntiResult";
```

Comment:
I'm once again promoting the "Object.assign" approach for namespacing the components, as used in the NewTabs.